### ISSUE TITLE
Basalt_filterSz_7_filters.npy

### ISSUE THREAD

**Issue by xiongzhu666** [TouqeerAhmad/skyline_detection#1]

**Comment by xiongzhu666** [comment]
Thanks for your good research
but where is the asalt_filterSz_7_filters.npy under  the filterBank file folder?

**Comment by TouqeerAhmad** [comment]
The filterBank folder would contain the trained filter file once you run the training script. The output of the training script is the filter file., that gets saved in the filterBank directory.

**Comment by xiongzhu666** [comment]
> The filterBank folder would contain the trained filter file once you run the training script. The output of the training script is the filter file., that gets saved in the filterBank directory.
Hi, thanks for your reply. 
I run the train and test script in CH1 datasets , but I just get some edge detection result in output folder as below. I did‘t get the skyline detection result. what else did I ignore??
![image](https://user-images.githubusercontent.com/52955431/153561780-0b586637-44a6-44a5-9084-5615bf47966f.png)

**Comment by TouqeerAhmad** [comment]
Yes, thank you for pointing this out, the repo is missing the Matlab code for running the dynamic programming on top of the mat file being saved in test.py at line 70. I shall push the required code over this weekend.

**Comment by Stedll** [comment]
Any update on that last push?

**Comment by Ch2ri1s** [comment]
Hello, you have not provided a matlab file to run the final effect. Can you provide it to us for learning？
